docs/dev_guide.rst

```diff
@@ -44,6 +44,9 @@ will run checks on the code style using precommit hooks.
 ``pytest``
 ==========
 
+.. warning::
+    Merging of the PR #55, it is expected that the ``pytest`` is going to fail. This is beacuse the ``pytest`` functions is testing the beta version of the API.
```

Comment:
```suggestion
```
This isn't true. You can just pass the beta url with pytest.

Reply:
What do we need to update in dev_guide?

Reply:
You need to say that they need to set the environment variable to be the beta URL as they run pytest.